locales/en/index.yml

```
@@ -3278,6 +3278,10 @@ idpay:
       validThrough: "Valid through {{endDate}}"
   initiative:
     configuration: 
+      cardLabels:
```

```suggestion
      instrumentStatus:
```
This brings more context